docs/searching-data.rst

```diff
@@ -0,0 +1,100 @@
+
+.. image:: ./_images/code-snippet-select-all.PNG
+  :align: center
+  :alt: Code snippet select all
+  
+Search
+======
+
+
+As described in the `Data Series Definition <./data-series-definition.rst>`_ page, a data series in Gro is a unique combination of the entities: item, metric, region, partner_region (optional), frequency, and source. To find the specific entity you would like to retrieve data for, you can use a variety of search methods. For example, :meth:`api.client.gro_client.GroClient.search` will return a list of IDs that match your search term. If you want to understand the differences between various search results, you may find the :meth:`api.client.gro_client.GroClient.search_and_lookup` method more helpful.
+
+:code:`client.search_and_lookup('items','Corn')` will yield a list of all items that contain "corn" in their name, along with supporting information like id, name, and ids of other items contained by a given item.
+
+Note: the above query will return a `generator <https://wiki.python.org/moin/Generators>`_ object. Running a query like the one below will print every result:
+
+
+Get data series
+===============
+
+Instead of searching for all the individual entity IDs required to create a data series, the :meth:`api.client.gro_client.GroClient.get_data_series` method will return a list of all the data series available for the filters you have supplied. For example, if you are interested in Russian Oats you could use the following code to find out all the available data series that have "Oats" (item_id = 327) as the item and "Russia" (region_id = 1168) as the region:
+To look up all descendants of region 1029 (Brazil) that are of level 4 (provinces):
+::
+
+  from api.client.gro_client.GroClient import REGION_LEVELS
+  provinces_of_brazil = client.get_descendant_regions(1029, REGION_LEVELS['province'])
+
+
+Lookup contains
+===============
+Our ontology is defined in terms of a graph, with metrics/items/regions containing others. In each case, you can see the :code:`'contains'` property in the output of :code:`client.lookup(type, id)`. For example:
+::
+
+  client.lookup('items', 10009)['contains']
+
+will return a list of items ids for items that are cereals (item_id = 10009): :code:`[..., 274, 422, ...]`. Once you have those ids, you can use the :meth:`api.client.gro_client.GroClient.lookup` function on each one to find more info, like their names, e.g.: :code:`client.lookup('items', 274)['name']` will return `Corn`.
+
+
+Get descendants
+===============
+
+Using the :code:`lookup()` method, you can get an entity's list of direct children (i.e. country->provinces). However, you may want all of the lower level regions that belong to a higher level region (i.e. country->provinces, districts, coordinates, etc.). To do this, there's a helper function which also gives the option of filtering by region level: :meth:`api.client.gro_client.GroClient.get_descendant_regions`
```

Comment:
super random - I saw it's rendering `>` in second line, maybe we should add space before & after -> so it doesn't look like a typo? 
<img width="705" alt="Screen Shot 2020-02-27 at 10 00 34 PM" src="https://user-images.githubusercontent.com/1120047/75491561-0b7acc00-59ae-11ea-84d1-b1f9201497aa.png">

Reply:
good catch! why not use arrows :D 
![image](https://user-images.githubusercontent.com/54555734/75494536-62839f80-59b4-11ea-88ce-7319ae46a811.png)